yb-voyager/src/srcdb/common.go

```diff
@@ -82,3 +83,43 @@ func processImportDirectives(fileName string) error {
 	}
 	return nil
 }
+// Strip the schema name from the names qualified by the source schema.
+//
+// ora2pg exports SYNONYM objects as VIEWs. The associated CREATE VIEW DDL statements
+// appear in the EXPORT_DIR/schema/synonyms/synonym.sql.
+//
+// The problem is that the view names are qualified with the Oracle's schema name.
+// Unless, the user is importing the schema in an exactly similarly named schema
+// on the target, the DDL statement will fail to import.
+//
+// The following function goes through all the names from the file and replaces
+// all occurrences of `sourceSchemaName.objectName` with just `objectName`.
+func stmtsToStripSourceSchemaNames(fileName string, sourceSchema string) error {
```

Comment:
Drop the `stmtsTo` part. It makes no sense.

Reply:
I missed that.